### ISSUE TITLE
Format numeric

### ISSUE THREAD

**Issue by iaguedo** [willdale/SwiftUICharts#130]
Very good graphics, in the data of the axes, you can format the numbers so that they come out with a comma and a period.
In the example photo I see that you put the comma, but I can't see how it does it.
Ejm: 10000.00 which is 10,000.00

**Comment by willdale** [comment]
Good point, I'll look into it.

**Comment by iaguedo** [comment]
Thank you

**Comment by willdale** [comment]
This has been fixed in Release 2.9.4.

A new parameter has been added to `yAxisLabels`

```swift
.yAxisLabels(...,
             formatter: NumberFormatter?,
             ...)
```

```swift
private var numberFormatter: NumberFormatter {
    let formatter = NumberFormatter()
    formatter.generatesDecimalNumbers = true
    formatter.numberStyle = .decimal
    formatter.minimumFractionDigits = 2
    formatter.maximumFractionDigits = 2
    return formatter
}
```

This allows for much greater flexibility in the numeric y axis labels. 

It overrides the `specifier` parameter.

Thanks,
Will

**Comment by iaguedo** [comment]
Hi, when I use formatter, the Ylabels come out on top of the first xlabel.
When I use touchOverlay the displayed amount could also be formatted.
Thank you!!

![20211106_115026](https://user-images.githubusercontent.com/16063899/140617554-9bb27c30-594d-4724-a793-422d95d47380.jpg)

**Comment by iaguedo** [comment]
I apologize, my mistake, I was putting the ylabel before the xlabel.
I would only have the doubt of formatting the value shown when making touch

**Comment by willdale** [comment]
This has been fixed in v2.9.5

**Comment by iaguedo** [comment]
Thank you so much
